alpha/alpha-server/src/main/java/org/apache/servicecomb/pack/alpha/server/GrpcStartable.java

```diff
@@ -103,9 +109,9 @@ public GrpcServerConfig getGrpcServerConfig() {
   private ServerBuilder getServerBuilder(int port) {
     return NettyServerBuilder.forAddress(
         new InetSocketAddress(serverConfig.getHost(), port))
-        .channelType(NioServerSocketChannel.class)
-        .bossEventLoopGroup(new NioEventLoopGroup(1))
-        .workerEventLoopGroup(new NioEventLoopGroup());
+        .channelType(selectorServerChannel())
+        .bossEventLoopGroup(selectorEventLoopGroup(1))
+        .workerEventLoopGroup(selectorEventLoopGroup(0));
```

Comment:
why is it selectorEventLoopGroup(0) here ? it will be default to match the num of the CPU cores ?

Reply:
If the number of threads is set to 0, the use default value DEFAULT_EVENT_LOOP_THREADS = Math.max(1, SystemPropertyUtil.getInt("io.netty.eventLoopThreads", NettyRuntime.availableProcessors() * 2));

Reply:
OK, thanks for explanation.